h/schemas/admin_group.py

```diff
@@ -22,17 +22,23 @@
 )
 
 
-def user_exists_validator_factory(user_svc):
-    def user_exists_validator(form, value):
-        user = user_svc.fetch(value['creator'], value['authority'])
+@colander.deferred
+def group_creator_validator(node, kw):
+    user_svc = kw['user_svc']
+    orgs = kw['organizations']
+
+    def validate(form, value):
```

Comment:
Perhaps add a comment or docstring here detailing what the validation is validating? Is it validating that the user exists at the authority associated with the organization selected in the form? (I think that's what's going on!)